code.gs

```diff
@@ -32,14 +32,14 @@ var myEmail = 'insertyouremailhere@someemail.com';
  * If you leave it blank it will default to your Google account name.
  * Note: this may not work when using a Gmail address sending emails to itself.
  */
-var emailSenderName = 'Birthday Notifications';
+var emailSenderName = 'Contacts Events Notifications';
 
 /*
- * ID OF THE BIRTHDAY CALENDAR
+ * ID OF THE CONTACTS EVENTS CALENDAR
  *
- * Open up https://calendar.google.com, in the menu on the left click on the arrow next to the birthday calendar
```

Comment:
I think I might have edited It as well...  
I'll try and have a look with my backup account tomorrow morning.